kdash: add page (#17154)
tldr-pages/tldr: pages/common/kdash.md

# kdash

> A simple terminal dashboard for Kubernetes.

> > Mode information: <https://github.com/kdash-rs/kdash/>.

Reviewer: ```suggestion
> Mode information: <https://github.com/kdash-rs/kdash/#usage>.
```
Quite a lot of scrolling to get to the usage instructions. Consider this.

- Show dashboard:

`kdash`

> - Show dashboard in debug mode and write logs:

Reviewer 2: https://github.com/kdash-rs/kdash/?tab=readme-ov-file#flags
> `-d, --debug[=<debug>]`: Enables debug mode and writes logs to `kdash-debug-<timestamp>.log` file in the current directory.
```suggestion
- Show dashboard in debug mode and write logs to a file in the current directory:
```

`kdash -d`

- Set the tick rate:

> `kdash -t 100`

Reviewer 3: ```suggestion
`kdash {{[-t|--tick-rate]}} 100`
```

> - Set the polling rate:

Reviewer 3: ```suggestion
- Set the polling rate (must be multiples of the tick rate):
```

> `kdash -t {{200}} -p {{400}}`

Reviewer: ```suggestion
`kdash {{[-t|--tick-rate]}} {{200}} {{[-p|--poll-rate]}} {{400}}`
```

> - Set the polling rate(must be multiples of the tick rate):

Reviewer: ```suggestion
- Set the polling rate (must be a multiple of the tick rate):
```
